.pre-commit-config.yaml

```diff
@@ -27,7 +27,7 @@ repos:
       - id: flake8
         entry: pflake8
         # We use a custom version of flake8 that includes a fix for Python 3.12 compatibility
```

Comment:
We could also get rid of this comment, right?

Reply:
Ohh, forgot about it :D Thanks!